lib/common.sh

```diff
@@ -945,11 +945,15 @@ common::set_cloud_binaries () {
 
   if [[ -x "$GSUTIL" && -x "$GCLOUD" ]]; then
     logecho -r $OK
-    return 0
   else
     logecho -r $FAILED
     return 1
   fi
+
+  # 'gcloud docker' access is now set in .docker/config.json
+  logrun $GCLOUD --quiet auth configure-docker || return 1
```

Comment:
this appears to be failing in CI @krzyzacy

Reply:
the gcloud in CI is probably too old.

Reply:
we should probably have a way to canary these changes and/or not have kubetest always use this repo from head..

Reply:
@david-mcmahon @ixdy

Reply:
talked offline, turns out that `docker version` fails with a non-zero exit code when it cant communicate with`dockerd`